### PR TITLE
ref(eventstream): Rename "relay" to "post-process-forwarder"

### PR DESCRIPTION
**Pull request by tkaemming**

### CODE REVIEW

File: src/sentry/runner/commands/run.py [hunk 1]
```diff
@@ -253,32 +253,45 @@ def cron(**options):
...
-            'The configured event stream backend does not need a relay '
-            'process to enqueue post-processing tasks. Exiting...\n')
-        return
+def _make_forwarder_command():
+    # XXX: Calling ``run.command`` mutates the option specifications for some
+    # reason that I don't care to identify (they only get picked up for the
+    # first registered command), so we have to create two distinct instances of
+    # this function for the temporary "relay" alas to work correctly. After the
+    # alias is removed, this hack can be removed and the task function can be
+    # defined at module level like everything else is normally.
```
Comment (tkaemming) [inline src/sentry/runner/commands/run.py]: 🤷‍♂️

**Review by tkaemming — COMMENTED** [review]
No longer" referenced as "relay" in `sentry.eventstream`:

```
(tmp-2756f5fefd0edb47) ted@veneno % git grep -iw relay -- src/sentry/eventstream
```

No longer referenced as "relay" (except for aliasing) in the runner:

```
(tmp-2756f5fefd0edb47) ted@veneno % git grep -iw relay -- src/sentry/runner
src/sentry/runner/commands/run.py:    # this function for the temporary "relay" alas to work correctly. After the
src/sentry/runner/commands/run.py:run.command('relay')(_make_forwarder_command())  # temporary alias for compatibility
```

Commands are still registered as both `relay` and `post-process-forwarder` (for now):

```
(tmp-2756f5fefd0edb47) ted@veneno % sentry run relay --help
Usage: sentry run relay [OPTIONS]

Options:
  --consumer-group TEXT           Consumer group used to track event offsets that have been
                                  enqueued for post-processing.
  --commit-log-topic TEXT         Topic that the Snuba writer is publishing its committed offsets
                                  to.
  --synchronize-commit-group TEXT
                                  Consumer group that the Snuba writer is committing its offset
                                  as.
  --commit-batch-size INTEGER     How many messages to process (may or may not result in an
                                  enqueued task) before committing offsets.
  --initial-offset-reset [earliest|latest]
                                  Position in the commit log topic to begin reading from when no
                                  prior offset has been recorded.
  -l, --loglevel [DEBUG|INFO|WARNING|ERROR|CRITICAL|FATAL]
                                  Global logging level. Use wisely.
  --logformat [human|machine]     Log line format.
  --help                          Show this message and exit.
(tmp-2756f5fefd0edb47) ted@veneno % sentry run post-process-forwarder --help
Usage: sentry run post-process-forwarder [OPTIONS]

Options:
  --consumer-group TEXT           Consumer group used to track event offsets that have been
                                  enqueued for post-processing.
  --commit-log-topic TEXT         Topic that the Snuba writer is publishing its committed offsets
                                  to.
  --synchronize-commit-group TEXT
                                  Consumer group that the Snuba writer is committing its offset
                                  as.
  --commit-batch-size INTEGER     How many messages to process (may or may not result in an
                                  enqueued task) before committing offsets.
  --initial-offset-reset [earliest|latest]
                                  Position in the commit log topic to begin reading from when no
                                  prior offset has been recorded.
  -l, --loglevel [DEBUG|INFO|WARNING|ERROR|CRITICAL|FATAL]
                                  Global logging level. Use wisely.
  --logformat [human|machine]     Log line format.
  --help                          Show this message and exit.
```

**Comment by mitsuhiko** [comment]
@tonyo update your graphs!

**Review by mattrobenolt — APPROVED** [review]
click it

**Pull request merged**